### DISCUSSION TITLE
The compiler cannot find the file you want to include. Make sure that you have uploaded the file and specified the file location correctly.

### DISCUSSION THREAD

**Discussion by TronYY** [ustctug/ustcthesis#319] [Q&A]
## 检查
- [x] 已经阅读《研究生学位论文撰写手册》或《关于本科毕业论文（设计）格式和统一封面的通知》
- [x] 已经将模板升级到了最新版
- [ ] 已经阅读使用说明文档 `ustcthesis.pdf`
- [ ] 已经阅读 [常见问题](https://github.com/ustctug/ustcthesis/wiki/常见问题)

## 编译环境
- 编译的系统：Ubuntu 18.04/Firefox/overleaf
- TeX 发行版：XeLatex
- 模板类型： master 

## 描述问题
在macos莫哈维可以正常产生pdf，但是各平台overleaf上均不行
Type X to quit or <RETURN> to proceed,
or enter new name. (Default extension: cls)

Enter file name: 
! Emergency stop.
<read *> 
   
l.5 ...tclass[degree=master]{ustcthesis}[2020/01/01]
                                                  
*** (cannot \read from terminal in nonstop modes)

The compiler cannot find the file you want to include. Make sure that you have uploaded the file and specified the file location correctly.
Learn more

截图：

![图片](https://user-images.githubusercontent.com/24478375/132154880-f4d44c2c-21ff-4642-b6da-8a45e79f3f80.png)

![图片](https://user-images.githubusercontent.com/24478375/132154824-db45cfc2-876b-4e1e-a344-cac430b784ed.png)

![图片](https://user-images.githubusercontent.com/24478375/132154712-45b53af3-bbe9-4ff1-b656-7d4412769c86.png)

![图片](https://user-images.githubusercontent.com/24478375/132154758-e3ebe659-0e58-4ad2-a321-1136dfcfdc38.png)

复现上述问题的代码：
```TeX
% !TeX encoding = UTF-8
% !TeX program = xelatex
% !TeX spellcheck = en_US
\documentclass[degree=master]{ustcthesis}[2020/01/01]

% degree    = doctor|master|bachelor
% language  = chinese|english

% 加载宏包、全部的配置
\input{ustcsetup.tex}
\begin{document}
```

**Comment by sikouhjw** [comment]
用 https://www.overleaf.com/latex/templates/latex-template-for-ustc-thesis/jvtdfkqsrycm 这里的呢？
我测试没有问题，Overleaf 上的模板过时了还是怎么，为什么不直接用配置好的？

**Comment by zepinglee** [comment]
"ustcthesis.cls" not found. 需要把 `.cls` 文件传上去。
